src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/ReactRedux-CSharp/ClientApp/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "allowJs": true,
+    "skipLibCheck": false,
```

Comment:
I think `skipLibCheck: false` is the default.  If I'm right, this line can be removed.

Reply:
skipLibCheck is re-added once the project is built.  Should I still remove?